qualang_tools/digital_filters/filters.py

```diff
@@ -89,16 +89,44 @@ def calc_filter_taps(
     return _check_hardware_limitation(qop_version, feedforward_taps, list(feedback_taps))
 
 
-def exponential_decay(x, a, t):
+def exponential_decay(x, a, t, s):
```

Comment:
I would make s=1 by Default, maybe it optional and maintaining backward compatibility